Week2/MAKEME.md

```diff
@@ -100,7 +100,7 @@ Copy the code in your `.js` file and run it. You will see that you will get a Sy
 5\.6 `console.log` the highest value.  
 
 6\. *Arrays*  
-6\.1 Declare an empty array. Make sure that the name you choose indicates 'plurality', because an array is capable of containing more than one element. (See [Naming conventions](../fundamentals/naming_conventions.md)).  
+6\.1 Declare an empty array. Make sure that the name you choose indicates 'plurality', because an array is capable of containing more than one element. (See [Naming conventions](https://github.com/HackYourFuture/fundamentals/blob/a4400bca1371d047fb3e186a6bfc1943259b6a5a/fundamentals/naming_conventions.md)).  
```

Comment:
Why does this link contain a commit hash? I think the link can just be:

https://github.com/HackYourFuture/fundamentals/blob/master/fundamentals/naming_conventions.md